### PR TITLE
7alip | Homework 2

### PR DESCRIPTION
**Pull request by 7alip**

### CODE REVIEW

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const earnings = tasks
+    .filter(task => task.duration >= 120) // <120 min (<2 hours)
+    .map(task => task.duration / 60)
+    .reduce((total, durations) => total + durations * hourlyRate, 0);
```
Comment (hasan-sh) [inline Week2/homework/maartjes-work.js]: Great that you used reduce :100:

File: Week2/homework/map-filter.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(item => item % 2 !== 0).map(odd => odd * 2);
```
Comment (hasan-sh) [inline Week2/homework/map-filter.js]: Perfect, also with your namings as well :)

File: Week2/homework/maartjes-work.js [hunk 3]
```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const earnings = tasks
+    .filter(task => task.duration >= 120) // <120 min (<2 hours)
```
[the remark this answers is not in the document]
Comment (7alip) [reply Week2/homework/maartjes-work.js]:
```javascript
function computeEarnings(tasks, hourlyRate) {
  const earnings = tasks
    .map(task => task.duration / 60)
    .filter(task => task.duration >= 2)
    .reduce((total, durations) => total + durations * hourlyRate, 0);

  return earnings;
}
```

I was trying the code above for solution but I got €0.00 as a result. That's why I've tried filtering first in order to pass the test. Not I got my mistake: 

```javascript
.filter(task => task.duration >= 2)
```

it should be:

```javascript
.filter(duration => duration >= 2)
```

[the remark this answers is not in the document]
Comment (hasan-sh) [reply Week2/homework/maartjes-work.js]: Exactly, did the test pass?

[the remark this answers is not in the document]
Comment (7alip) [reply Week2/homework/maartjes-work.js]: Yes, it passed also

**Pull request closed without merging**